### PR TITLE
Add documentation to timeseriesChart

### PR DESCRIPTION
**Pull request by follan**

### CODE REVIEW

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-151.gearboxjs.preview.cogniteapp.com

File: src/components/TimeseriesChart/stories/full.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Timeseries Chart
+
+<!-- STORY -->
+
+### Description:
+
+This component loads the datapoints given a timeseries id and renders a line chart of those points.
+The component requires only `timeseriesIds`
+
+####
```
Comment (mrtraser) [inline src/components/TimeseriesChart/stories/full.md]: Guess word **Usage** is should be here )

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-151.gearboxjs.preview.cogniteapp.com

File: src/components/TimeseriesChart/stories/full.md (hunk 1 above)
Comment (follan) [reply src/components/TimeseriesChart/stories/full.md]: Fixed

**Pull request merged**